VERSION

```diff
@@ -1 +1 @@
-3.0.2
```

Comment:
Why 3.1.1 and not 3.1.0?

Reply:
Ok, I think I see that this is in prep for the future version.

Reply:
right, we missed the version number change step in v3.1.0, and we wrote the known bug on wiki:
https://wiki.edgexfoundry.org/pages/viewpage.action?pageId=88212172#Napa(NextRelease)-KnownBugs
it only implacts the logs, and it should be a blocker for release
it's the last one we need to manually change the version number.

Reply:
> it only implacts the logs, and it should be a blocker for release

@cloudxxx8 , I assume you mean "it should **not** be a blocker for release", which I agree.

Reply:
right, should **not**